src/mono/sample/wasm/blazor-frame/blazor.csproj

```diff
@@ -10,8 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="8.0.0-rc.1.23414.10" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="8.0.0-rc.1.23414.10" PrivateAssets="all" />
```

Comment:
Two remaining pre-release hits:

```
eng/testing/workloads-testing.targets:    <!--<AdditionalSharedFrameworkToInstallArguments Include="-Version 8.0.0-rc.2.23457.7" />-->
src/mono/wasm/Wasm.Build.Tests/WorkloadTests.cs:            // extract pack names from ./artifacts/bin/dotnet-latest/packs/Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.linux-x64/8.0.0-rtm.23470.1/data/UnixFilePermissions.xml
```

Reply:
the second one is fine since it's an example in a comment, we have plenty of those